server/models/Profile.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+
+const dayAvalSchema = new mongoose.Schema({
+    from: String,
```

Comment:
I presume the 'from/to' fields here are for time slots? In that case maybe we can call them startTime/endTime ?